plugins/shortcuts/src/ShortcutItem.tsx

```diff
@@ -58,9 +58,10 @@ const getIconText = (title: string) =>
 type Props = {
   shortcut: Shortcut;
   api: ShortcutApi;
+  allowExternalLinks?: boolean;
 };
 
-export const ShortcutItem = ({ shortcut, api }: Props) => {
+export const ShortcutItem = ({ shortcut, api, allowExternalLinks }: Props) => {
```

Comment:
Will the existing `SidebarItem` handle external links correctly? 🤔

Reply:
Yeah, you can test it in the Uffizzi environment